src/dbnode/server/server.go

```diff
@@ -1202,8 +1202,11 @@ func withEncodingAndPoolingOptions(
 	postingsListOpts := poolOptions(policy.PostingsListPool, scope.SubScope("postingslist-pool"))
 	postingsList := postings.NewPool(postingsListOpts, roaring.NewPostingsList)
 
-	resultsPool := index.NewResultsPool(
-		poolOptions(policy.IndexResultsPool, scope.SubScope("index-results-pool")))
+	// Need to actually set pools
```

Comment:
nit: Remove this now that the pools are set?

Reply:
Good call